apidocs/getTrytes.md

```diff
@@ -0,0 +1,111 @@
+ See utility functions for more details.
+
+<Tabs> 
+
+<Tab language="Python">
+
+<Section type="request">
+import urllib2
+import json
+
+command = {"command": "getTrytes", "hashes": ["P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999", "P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999"]}
+
+stringified = json.dumps(command)
+
+headers = {
+    'content-type': 'application/json',
+    'X-IOTA-API-Version': '1'
+}
+
+request = urllib2.Request(url="http://localhost:14265", data=stringified, headers=headers)
+returnData = urllib2.urlopen(request).read()
+
+jsonData = json.loads(returnData)
+
+print jsonData
+</Section>
+
+<Section type="response">
+{"duration": "166", "trytes": ["P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999", "P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999"]}
+</Section>
+
+<Section type="error">
+{"error": "'command' parameter has not been specified"}
+</Section>
+
+<Tab language="NodeJS">
+
+<Section type="request">
+var request = require('request');
+
+var command = {"command": "getTrytes", "hashes": ["P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999", "P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999"]}
+
+var options = {
+  url: 'http://localhost:14265',
+  method: 'POST',
+  headers: {
+    'Content-Type': 'application/json',
+		'X-IOTA-API-Version': '1',
+    'Content-Length': Buffer.byteLength(JSON.stringify(command))
+  },
+  json: command
+};
+
+request(options, function (error, response, data) {
+  if (!error && response.statusCode == 200) {
+    console.log(data);
+  }
+});
+</Section>
+
+<Section type="response">
+{"duration": "629", "trytes": ["P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999", "P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999"]}
+</Section>
+
+<Section type="error">
+{"error": "'command' parameter has not been specified"}
+</Section>
+
+<Tab language="cURL">
+
+<Section type="request">
+curl http://localhost:14265 
+-X POST 
+-H 'Content-Type: application/json' 
+-H 'X-IOTA-API-Version: 1' 
+-d '{"command": "getTrytes", "hashes": ["P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999", "P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999"]}'
+</Section>
+
+<Section type="response">
+{"duration": "944", "trytes": ["P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999", "P9KFSJVGSPLXAEBJSHWFZLGP9GGJTIO9YITDEHATDTGAFLPLBZ9FOFWWTKMAZXZHFGQHUOXLXUALY9999"]}
+</Section>
+
+<Section type="error">
+{"error": "'command' parameter has not been specified"}
+</Section>
+</Tabs<
+
+
+
+***
+	
+|Parameters | Description |
+|--|--|
+| hashes | List of transaction hashes of which you want to get trytes from. |
```

Comment:
List of transaction hashes of which you want to get trytes from.
-> List of transaction hashes you want to get trytes from.